runatlantis.io/docs/using-atlantis.md

```diff
@@ -1,19 +1,29 @@
 # Using Atlantis
 
-Atlantis currently supports three commands that can be run via pull request comments:
+Atlantis handle commands via pull request comments.
+![Help Command](./images/pr-comment-help.png)
+Currently supports following commands.
 [[toc]]
 
+---
 ## atlantis help
-![Help Command](./images/pr-comment-help.png)
 ```bash
 atlantis help
 ```
 ### Explanation
 View help
 
+---
+## atlantis version
+```bash
+atlantis version
+```
+
+### Explanation
+Print the output of 'terraform version'.
+
 ---
 ## atlantis plan
-![Plan Command](./images/pr-comment-plan.png)
```

Comment:
It's difficult to capture same image for other commands. So just show one image on the page top.